src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/utils/flowhistory/FlowHistoryRangeConstraints.java

```diff
@@ -66,7 +73,7 @@ ErrorType.PARAMETERS_INVALID, format("Invalid `max_count` argument '%s'.",
 
         contentRangeRequiredPredicate = size ->
                 (!maxCountInput.isPresent() && !timeFromInput.isPresent() && !timeToInput.isPresent()
-                        && size == DEFAULT_MAX_HISTORY_RECORD_COUNT);
+                        && size > DEFAULT_MAX_HISTORY_RECORD_COUNT);
```

Comment:
Won't this change our API behavior?

Reply:
It will, it's been an unnoticed typo. The paging is needed once there are more records than a number of records per page. Now I added a unit test for this case as well.